suite-native/atoms/src/Sheet/BottomSheetHeader.tsx

```diff
@@ -1,48 +1,46 @@
 import React from 'react';
-import { TouchableOpacity, View } from 'react-native';
+import { View } from 'react-native';
 
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Icon } from '@trezor/icons';
 
 import { Text } from '../Text';
+import { IconButton } from '../Button/IconButton';
 
 type BottomSheetHeaderProps = {
     title?: string;
+    subtitle?: string;
     onCloseSheet: () => void;
 };
 
-const CLOSE_BUTTON_SIZE = 40;
-
-const closeButtonStyle = prepareNativeStyle(utils => ({
-    borderRadius: utils.borders.radii.round,
-    height: CLOSE_BUTTON_SIZE,
-    width: CLOSE_BUTTON_SIZE,
-    justifyContent: 'center',
-    alignItems: 'center',
-}));
-
 const sheetHeaderStyle = prepareNativeStyle(utils => ({
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
-    paddingHorizontal: utils.spacings.medium,
-    paddingVertical: utils.spacings.medium,
-}));
-
-// To center the title
-const closeButtonPlaceholderStyle = prepareNativeStyle(() => ({
-    width: CLOSE_BUTTON_SIZE,
+    paddingHorizontal: utils.spacings.large,
+    paddingVertical: utils.spacings.extraLarge,
+    borderBottomWidth: 1,
+    borderBottomColor: utils.colors.gray300,
```

Comment:
There is a Divider component in Atoms package which could be used for this

Reply:
done